src/controllers/__tests__/project.test.ts

```diff
@@ -1,51 +1,79 @@
 import { build } from "../../../tests/helper";
+import { createTestProject } from "../../../tests/helper/utils";
+import { Project } from "../../entities";
 
 const app = build();
 
-describe("/project/:projectId", () => {
-  test("Pass proper body - it should successfully edit existing project", async () => {
-    const mockName = "this is mocked name";
-    const res = await app.inject({
-      method: "put",
-      url: "/project/1",
-      payload: {
-        name: mockName,
-      },
+describe("/projects/:projectId", () => {
+  describe("get project", () => {
+    let mockProject: Project;
+
+    beforeAll(async () => {
+      mockProject = await createTestProject();
     });
 
-    expect(res.statusCode).toEqual(200);
-    expect(res.body).toContain(mockName);
-  });
+    afterAll(async () => {
+      await mockProject.remove();
+    });
 
-  test("Pass broken query - it should return error that indicates project not found", async () => {
-    const expectedErrorMessage =
-      '{"statusCode":500,"error":"Internal Server Error","message":"Invalid project id: NaN"}';
-    const mockName = "test";
-    const res = await app.inject({
-      method: "put",
-      url: "/project/a",
-      payload: {
-        name: mockName,
-      },
+    test("Pass existing projectId - it should successfully fetch existing project", async () => {
+      const res = await app.inject({
+        method: "get",
+        url: `/project/${mockProject.id}`,
+        payload: {},
+      });
+
+      expect(res).toBe(mockProject);
     });
 
-    expect(res.statusCode).toEqual(500);
-    expect(res.body).toContain(expectedErrorMessage);
+    test.todo("Pass not existing projectId - it should successfully fetch existing project");
```

Comment:
I'm confused by the message. Is this for the existing one? not the one which doesn't

Reply:
Oops, I over looked

Reply:
It was todo for myself so I didn't care much grammar